src/3.x/extending.md

```diff
@@ -119,4 +120,55 @@ The autocomplete can contain as many Elasticsearch indexes as you wish. You can
 ],
 ```
 
-You can use `categories.blade.php` as an example for how to properly display new indexes in the autocomplete.
\ No newline at end of file
+### Configuration
+
+There are two ways to define a new additional index. The shorthand way is by only giving the index name and searchable fields, like so:
+
+```php
+'categories' => ['name^3', 'description'],
+```
+
+Alternatively, you can have more control by expanding the configuration like below:
+
+```php
+'blogs' => [
+    'fields' => ['title', 'tags'],
+    'size' => 3,
+    'stores' => ['my_second_store'],
+    'sort' => ['date' => 'desc'],
+],
+```
+
+- `fields` (required)  
+Here you can define the searched fields
+- `size`  
+This overrides the default `size` parameter for this index, letting you define a different page size for this specific index. In the above example, we only get a maximum of 3 blogs to be shown in the autocomplete instead of the default 10.
+- `stores`  
+Limits your index to specific stores. In the above example, the `blogs` index will only be queried on the store with the code `my_second_store`. This allows you to have store-specific indexes. The index will be used on all stores if this parameter is not defined.
+- `sort`  
+This lets you define an alternative sorting in case the default ES sorting doesn't suffice. See [the ES documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/sort-search-results.html) for how to use this parameter correctly.
```

Comment:
Is this the cleanest formatting? Maybe use a table? Or keep everything on one line, this will result in weird html. Or maybe a collapse to toggle the description?

Reply:
What's the standard we use elsewhere? I see a table for example here: https://docs.rapidez.io/3.x/testing.html#waituntiltrueforduration, so I think a table would be best.